Build fails on Windows environment

When building on Windows version f6817f853e0d0e4a811d34dd0e44d0b244a6f1c7 , the admin console build fails.
The problem is the following

`[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.1:yarn (admin console build) on project aac: Failed to run task: 'yarn dist' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]`

The problem might be an encoding end of line issue.

Note that a similar issue was probably fixed with commit 9af1f58329a9654ce09daeb29966492221ef4621

Will follow with fixing pull request

--- a/admin-console/.eslintrc
+++ b/admin-console/.eslintrc
@@ -11,7 +11,11 @@
     ],
     "rules": {
         "no-use-before-define": "off",
-        "prettier/prettier": "error",
+        "prettier/prettier": [
+            "error",{
+                "endOfLine": "auto"
+            }
+        ],
         "no-restricted-imports": [
             "error",
             {

